### PR TITLE
#150: added initial impl. of folder creation

### PR DESCRIPTION
**Pull request by ggajos**
for #150

### CODE REVIEW

**Comment by karato** [comment]
Many thanks for the PR, let me find a reviewer for it

**Comment by karato** [comment]
@carlosmiranda do this review

File: src/main/java/com/nerodesk/takes/doc/TsDir.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import org.takes.facets.fork.FkRegex;
+import org.takes.facets.fork.TsFork;
+import org.takes.tk.TkEmpty;
+
+/**
+ * Takes for a directories.
+ *
+ * @author Grzegorz Gajos (grzegorz.gajos@opentangerine.com)
+ * @version $Id$
+ * @since 0.3
```
Comment (carlosmiranda) [inline src/main/java/com/nerodesk/takes/doc/TsDir.java]: We already have version 0.3, maybe we should use 0.4.

File: src/test/java/com/nerodesk/takes/doc/TsDirTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.takes.rq.RqFake;
+
+/**
+ * Tests for {@code TsDir}.
+ *
+ * @author Grzegorz Gajos (grzegorz.gajos@opentangerine.com)
+ * @version $Id$
+ * @since 0.1
```
Comment (carlosmiranda) [inline src/test/java/com/nerodesk/takes/doc/TsDirTest.java]: Let's also use 0.4 version here.

File: src/main/java/com/nerodesk/takes/doc/TsDir.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import java.io.IOException;
+import org.takes.Request;
+import org.takes.Take;
+import org.takes.Takes;
+import org.takes.facets.fork.FkRegex;
+import org.takes.facets.fork.TsFork;
+import org.takes.tk.TkEmpty;
+
+/**
+ * Takes for a directories.
```
Comment (carlosmiranda) [inline src/main/java/com/nerodesk/takes/doc/TsDir.java]: Should be "Takes for a directory" (unless I'm misinterpreting the purpose of this class).

**Comment by carlosmiranda** [comment]
@ggajos A few minor comments above. Apologies for the delay in review.

**Comment by ggajos** [comment]
@carlosmiranda thanks for feedback, please have a look if everything is fine now

**Comment by carlosmiranda** [comment]
@rultor let's merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by ggajos** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by ggajos** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@rultor merge

**Comment by ghost** [comment]
@ggajos thanks.

**Comment by karato** [comment]
@carlosmiranda **17 mins** was added to your account, many thanks for your contribution! The task took 55100726.... extra minutes for review comments (c=2)... added +17 to your rating, now it is equal to [+5335](http://www.netbout.com/b/35023?open=rating)

**Comment by karato** [comment]
@rultor deploy

**Pull request merged**